test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -56,6 +57,15 @@ def test_ecs_pytorch_training_mnist_gpu(gpu_only, ecs_container_instance, pytorc
                                          num_gpus=num_gpus)
 
 
+def can_run_s3_plugin(framework_version):
+    return Version(framework_version) in SpecifierSet(">=1.7") and Version(framework_version) != Version("1.9.0")
+
+
+def validate_or_skip_s3_plugin(framework_version):
+    if not can_run_s3_plugin(framework_version):
+        pytest.skip("S3 plugin is not yet supported on 1.9.0")
```

Comment:
Please make this message more generic, as the plugin isn't supported on PT 1.6 either.

How about `S3 Plugin is not supported on this PyTorch version`